pom.xml

```diff
@@ -47,6 +54,12 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-library</artifactId>
+			<version>2.1</version>
```

Comment:
can you move the version to the properties like structured logging?

Reply:
Will do.